modules/core/server/controllers/core.server.controller.js

```diff
@@ -14,6 +14,7 @@ exports.renderIndex = function (req, res) {
       displayName: validator.escape(req.user.displayName),
       provider: validator.escape(req.user.provider),
       username: validator.escape(req.user.username),
+      _id: req.user._id,
```

Comment:
I wanted to keep the change simple. How about this?

`<a class="btn btn-primary" ng-click="vm.remove()" ng-if="vm.user.username !== vm.authentication.user.username">`

This should fix the issue + no extra code change needed except changing the _id to username.